docs/ilc_app_interface.md

```diff
@@ -59,4 +59,14 @@ See more information about the [lifecycle functions here](https://single-spa.js.
 * `domElementGetter(): HTMLElement` - returns ref to `HTMLElement` that should be used as container to render app's content
 * `getCurrentPathProps(): {}` - returns _Props_ that were defined for current path
 * `getCurrentBasePath(): string` - returns same value as `basePath` param in `routerProps` query parameter
-* `errorHandler(error, errorInfo = {}): void` - app MUST use it to propagate all unhandled errors
\ No newline at end of file
+* `errorHandler(error, errorInfo = {}): void` - app MUST use it to propagate all unhandled errors
+
+### Page transitions spinner
+You can set own spinner with help of your main html-template:
```

Comment:
with *the* help of

Reply:
- I think we need to move it to separate doc called "Animation during reroute" 
- This particular example will never work as is. Pls give a concrete one that users can copy/paste.

Reply:
https://github.com/namecheap/ilc/pull/143/commits/1606d29992c67cff54850a95c2b4face8fbfb99e